rotkehlchen/tests/db/test_trades.py

```diff
@@ -198,7 +198,7 @@ def test_query_trades_including_ammswaps(data_dir, username, sql_vm_instructions
             amount0_out=ZERO,
             amount1_out=FVal(3.2),
         ), AMMSwap(
-            tx_hash='0x5',
+            tx_hash=deserialize_evm_tx_hash('0x90f68af0ebbbb8d4938a4fbd07a70862e806124abd907d1225f25a10afda0185'),  # noqa: E501
```

Comment:
can you move these five hashes in separate vars in the start of the test and refer them with their end numbers as before?

Took me some time to realize you kept them as different hashes.